src/ontology/cl-edit.owl

```diff
@@ -30680,6 +30681,20 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29904965"^^xsd:string) o
 AnnotationAssertion(rdfs:label obo:CL_4032001 "reelin GABAergic cortical interneuron")
 EquivalentClasses(obo:CL_4032001 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000013879)))
 
+# Class: obo:CL_4033014 (peg cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22911892"^^xsd:string) obo:IAO_0000115 obo:CL_4033014 "A small, peg-like epithelial cell of the fallopian tube (oviduct) epithelium, with very little cytoplasm. These cells are much rarer than the ciliated and secretory epithelial cells of the fallopian tube epithelium and are often found intercalated between them, with high concentrations at the fimbriated, distal end of the tube. They may have a regenerative/stem-cell function. In humans, markers include EPCAM, CD44, and ITGA6."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033014 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033014 "2023-02-02T15:07:12Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22911892"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_4033014 "FTE stem-like cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22911892"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_4033014 "FTESC")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22911892"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_4033014 "fallopian tube epithelial stem-like cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22911892"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_4033014 "intercalary cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22911892"^^xsd:string) rdfs:comment obo:CL_4033014 "In vitro evidence supports a stem-cell/regenerative function for these cells"^^xsd:string)
+AnnotationAssertion(rdfs:label obo:CL_4033014 "peg cell"^^xsd:string)
+SubClassOf(obo:CL_4033014 obo:CL_0000066)
+SubClassOf(obo:CL_4033014 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0004804))
```

Comment:
The dbxref focuses on the fallopian tube. If this cell type is restricted to humans, this axiom seems OK since "mammalian oviduct" is an exact synonym. But note, Uberon partly defines the fallopian tube as a subdivision of the oviduct. 

Also consider that neighbouring cell types invoked in the text definition ('fallopian tube secretory epithelial cell' and 'fallopian tube ciliated cell') include the axiom 'part of' some 'fallopian tube'.

Reply:
Peg cells are not restricted in humans and there are some [references](https://pubmed.ncbi.nlm.nih.gov/30023181/) about it. However, the literature research has focused in humans. Therefore I could create "peg cell" and "fallopian tube peg cell" terms independently, but that is not very practical. It is a bit messier because in humans the oviduct receive a different names (fallopian tube), but creating two terms we are duplicating them. The main problem comes with the synonyms, as they refer to fallopian tubes, so they are only exact synonyms in humans, so probably a narrow synonym (or even broad) would be more appropriate.